ProcessManager.py

```diff
@@ -6,14 +6,24 @@
 from multiprocessing.context import Process # Processes are fully functional on Windows (as of August 20, 2017).
 import time
 import sys
+from subprocess import Popen
+
+class RoboProcesses:
+            def _init_(self, cmd):
+                self.cmd = cmd
+                self.set = None
+                Popen(self.set)
```

Comment:
The line `Popen(self.set)` tries to start a program `None` which won't work. This line could probably just be removed.

Reply:
Yeah, will do... Wasn't sure if I had to have Popen there or how I am suppose to set it to None... I guess just have Popen executed in startAllProcesses?